fastlane/Fastfile

```diff
@@ -68,27 +70,50 @@ lane :release do |options|
   end
 
   sh("git commit -m 'Bump #{version_number}'")
-  sh("git tag #{version_number}")
 
   if(!prompt(text: "Will push changes. All looking good?", boolean: true))
     UI.user_error!("Not pushing changes")
   end
 
-  push_to_git_remote(tags: true)
+  push_to_git_remote(tags: false)
+
+  # Open a new PR
+  create_pull_request(
+    api_token: ENV["GITHUB_TOKEN"],
+    repo: "GetStream/stream-chat-swift",
+    title: "#{version_number} Release",
+    head: "release/#{version_number}",
+    base: "main",
+    body: "#{changes}"
+)
 
+  UI.success("Successfully started release #{version_number}")
+end
+
+desc "Completes an SDK Release"
+lane :publish_release do |options|
+  version = options[:version]
   github_release = set_github_release(
```

Comment:
In here we should be able to call `make frameworks` (we can exclude static libs for now), which will create zip files in the Products folder. All the zip files should be uploaded to github as release artifacts.